### PR TITLE
Fixing accuracy formulas

### PR DESCRIPTION
**Pull request by cjames7223**
Just an update to the accuracy formulas. Seems to be more accurate with OSRS

### CODE REVIEW

**Review by reinismu — COMMENTED** [review]
How did you test it? What makes you think it is more accurate now?

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java [hunk 1]
```diff
@@ -309,7 +309,7 @@ private static float effectiveMagicLevel(Mobile entity) {
         if (fightStyle == FightStyle.ACCURATE)
             mag += 3;
         else if (fightStyle == FightStyle.DEFENSIVE)
-            mag += 1;
+            mag += 3;
```
Comment (MatthewBishop) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java:312]: Are these lines needed? https://github.com/RSPSApp/elvarg-rsps/pull/238/files#diff-70d3e98f731f79bd131aa1ed96cf47c15d3b09315769073b15e513f011c08ed7R308-R315 I dont think the style bonus has any impact on magic spell accuraccy.

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java [hunk 2]
```diff
@@ -322,7 +322,7 @@ public static int defenseMagicRoll(Mobile enemy) {
 
         int defRange = (enemy.isNpc() ? 0 : enemy.getAsPlayer().getBonusManager().getDefenceBonus()[BonusManager.DEFENCE_MAGIC]);
 
-        defLevel *= defRange + 64;
+        defLevel *= defRange + 3;
```
Comment (MatthewBishop) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java:325]: For player defense from magic attacks it is a combination of their defense level and also their magic level. This looks to only be taking into account their magic level.

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java [hunk 3]
```diff
@@ -56,7 +56,7 @@ public static float hitChance(int attRoll, int defRoll) {
     }
 
     private static float effectiveAttackLevel(Mobile entity) {
-        float att = 8;
+        float att = 10;
```
Comment (MatthewBishop) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java:59]: Why 10? This whole method needs to be reworked to use the correct order of operations. Boosted level should be multiplied by the pray bonus first and then rounded down. Then the style bonus added and then +8 before void calcs

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java [hunk 4]
```diff
@@ -229,7 +229,7 @@ public static int defenseRangedRoll(Mobile enemy) {
 
 
     private static float effectiveRangedAttack(Mobile entity) {
-        float rngStrength = 8;
+        float rngStrength = 9;
```
Comment (MatthewBishop) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java:232]: Why 9? This whole method needs to be reworked to use the correct order of operations. Boosted level should be multiplied by the pray bonus first and then rounded down. Then the style bonus added and then +8 before void calcs

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java [hunk 5]
```diff
@@ -279,7 +279,7 @@ public static int attackRangedRoll(Mobile entity) {
     // Magic
 
     private static float effectiveMagicLevel(Mobile entity) {
-        float mag = 8;
+        float mag = 9;
```
Comment (MatthewBishop) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/formula/AccuracyFormulasDpsCalc.java:282]: 8 is correct for players, 9 is correct for NPCS.

**Review by MatthewBishop — CHANGES_REQUESTED** [review]
Ive added some feedback to this PR.

**Pull request closed without merging**